activesupport/lib/active_support/core_ext/securerandom.rb

```diff
@@ -16,7 +16,7 @@ module SecureRandom
   #
   #   p SecureRandom.base58 # => "4kUgL2pdQMSCQtjE"
   #   p SecureRandom.base58(24) # => "77TMHrHJFvFDwodq8w7Ev2m7"
-  if RUBY_VERSION >= "3.3"
+  if RUBY_VERSION >= "3.3" && RUBY_RELEASE_DATE > "2023-05-12"
```

Comment:
```suggestion
  if RUBY_VERSION > "3.3.0.preview1"
```

Would this work as well? Because this would be fine

Reply:
@rafaelfranca `RUBY_VERSION` returns `3.3.0` when using  Ruby 3.3.0preview1